embedly/embedly/api/views.py

```diff
@@ -34,8 +34,8 @@ def version():
     )
 
 
-@blueprint.route('/extract')
-def extract_urls():
+@blueprint.route('/extract', methods=['GET'])
+def extract_urls_v1():
```

Comment:
Filed activity-stream specific bug at https://github.com/mozilla/activity-streams/issues/259 to remove any GET calls/logic to the proxy.